partner_compassion/wizards/partner_check_double.py

```diff
@@ -0,0 +1,39 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Fluckiger Nathan <nathan.fluckiger@hotmail.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from odoo import api, fields, models, _
+
+
+class PartnerCheckDouble(models.TransientModel):
+    _name = "res.partner.check.double"
+
+    newpartner_id = fields.Many2one('res.partner')
+    mergeable_partner_ids = fields.Many2many(
+        'res.partner', 'res_partner_check_double_relation', 'wizard_id',
+        'partner_id', readonly=True)
```

Comment:
add attribute `string="Similar partners found"` (it will be displayed as field name in the view)